roles/keycloak/tasks/blocks/configure_theme.yml

```diff
@@ -3,9 +3,14 @@
 - name: Acquire tokens
   include_tasks: blocks/helpers/get_tokens.yml
 
+## we should issue the configuration calls from all host machines locally (and not through the proxy),
+## because we want to be absolutely sure that all calls have been executed on all keycloak nodes at least once.
+- ansible.builtin.set_fact:
+    keycloak_local_host: "http://localhost:8080/{{ keycloak_base_url_path }}"
```

Comment:
I would suggest the following change:

- Use `keycloak_bind_port` (which already default to 8080) instead of hardcoding 8080 here 

Do we also need to introduce `keycloak_local_address` var ? 
Alternatively we can just define keycloak_local_host in `defaults/main.yml`:

```
keycloak_local_host: "http://localhost:{{ keycloak_bind_port }}/{{ keycloak_base_url_path }}"
```
It should be close to the definition of `keycloak_bind_address` because if this is not `0.0.0.0` then `localhost` would not work, right?

Reply:
> I would suggest the following change:
> 
> * Use `keycloak_bind_port` (which already default to 8080) instead of hardcoding 8080 here
> 
You are absolutely right!

> Do we also need to introduce `keycloak_local_address` var ? Alternatively we can just define keycloak_local_host in `defaults/main.yml`:
> 
> ```
> keycloak_local_host: "http://localhost:{{ keycloak_bind_port }}/{{ keycloak_base_url_path }}"
> ```
> 
> It should be close to the definition of `keycloak_bind_address` because if this is not `0.0.0.0` then `localhost` would not work, right?

Indeed, this could lead to failures. 
I'll introduce a `keycloak_local_address` variable in the defaults.
Anyway, everything should be customizable.

Reply:
There is already a keycloak_local_address, which is computed on the fly in the beginning (it's already used in the checks of the role about whether the keycloak service is up and running).
I'll use that variable (which is tested and works well on all environments).